packages/vite/bin/vite.js

```diff
@@ -19,7 +19,7 @@ const profileIndex = process.argv.indexOf('--profile')
 
 if (debugIndex > 0) {
   let value = process.argv[debugIndex + 1]
-  if (!value || value.startsWith('-')) {
+  if (!value || value.startsWith('-') || debugIndex + 1 === filterIndex) {
```

Comment:
Shouldn't `value.startsWith('-')` catch `debugIndex + 1 === filterIndex` already? What am I missing? 🤔

Reply:
Sorry, I seem to have made a stupid mistake. 😅

When I executed the above-mentioned command last night, filtering debug information didn't seem to take effect, but I just tried it again and it took effect again.

It looks like I did something wrong last night.